clang/lib/Sema/SPIRVBuiltins.td

```diff
@@ -897,15 +897,15 @@ foreach name = ["BitCount"] in {
 // 3.32.20. Barrier Instructions
 
 foreach name = ["ControlBarrier"] in {
-  // TODO: Allow enum flags instead of UInt ?
-  // TODO: We should enforce that the UInt must be a literal.
-  def : SPVBuiltin<name, [Void, UInt, UInt, UInt], Attr.Convergent>;
+  // TODO: Allow enum flags instead of Int ?
```

Comment:
Should this also have a clang test?

Reply:
updated clang test in https://github.com/intel/llvm/pull/17438/commits/49d96411e4e0ae75bb56182a5718d02348cab509